OpenRA.Mods.Common/Traits/Production.cs

```diff
@@ -49,20 +49,16 @@ void INotifyCreated.Created(Actor self)
 			building = self.TraitOrDefault<Building>();
 		}
 
-		public virtual void DoProduction(Actor self, ActorInfo producee, ExitInfo exitinfo, string factionVariant)
+		public virtual void DoProduction(Actor self, ActorInfo producee, ExitInfo exitinfo, TypeDictionary inits)
```

Comment:
Unrelated to this specific set of changes, but after wasting some time getting up to speed I feel this should be `protected virtual void ProduceInternal(...)`...